networking.md

```diff
@@ -24,17 +24,14 @@ Here we'll explain some of the mechanics that drive the Filecoin network layer.
 
 ## Peer discovery
 
-We use the libp2p Kademlia Distributed Hashtable (Kad DHT) service to:
+We use the [`go-libp2p-kad-dht`](https://github.com/libp2p/go-libp2p-kad-dht) service to:
 
-1. discover Filecoin peers to interface with upon boot.
-2. lookup the network addresses of a specific peer we need to interact with, starting from a peer ID, for the purposes of storage and retrieval.
+1. Discover Filecoin peers at start up.
+2. Lookup the network addresses of a specific peer ID, for the purposes of Filecoin-related storage and retrieval.
 
```

Comment:
Should we also include autorelay provider record search / provide here?  We currently only mention this dht use in the autorelay section.  No strong opinions just pointing out.  cc @anorth

Reply:
@ZenGround0 could you make an edit to that effect in another PR? The libp2p people don't necessarily know how we use it.